test/api_integration/apis/custom_integration/integrations.ts

```diff
@@ -25,7 +25,7 @@ export default function ({ getService }: FtrProviderContext) {
 
         expect(resp.body).to.be.an('array');
 
-        expect(resp.body.length).to.be(56);
+        expect(resp.body.length).to.be(55);
```

Comment:
This total was reduced because of the removal of the App Search JSON integration.